src/main/java/it/gov/pagopa/payhub/activities/service/debtposition/custom/fine/DebtPositionFineValidation.java

```diff
@@ -0,0 +1,32 @@
+package it.gov.pagopa.payhub.activities.service.debtposition.custom.fine;
+
+import it.gov.pagopa.pu.debtposition.dto.generated.DebtPositionDTO;
+import it.gov.pagopa.pu.debtposition.dto.generated.PaymentOptionDTO;
+import it.gov.pagopa.pu.debtposition.dto.generated.PaymentOptionStatus;
+import it.gov.pagopa.pu.debtposition.dto.generated.PaymentOptionTypeEnum;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Lazy
+@Service
+public class DebtPositionFineValidation {
+
+    public boolean validateFine(DebtPositionDTO debtPositionDTO) {
```

Comment:
ps: create inner private method for each validation (PO counts) (PO types) in order to improve the readability of this method and understand what kind of validation is verifying